control/tap-ctl-spawn.c

```diff
@@ -196,5 +232,15 @@ tap_ctl_spawn(void)
 	if (id < 0)
 		EPRINTF("get_id failed, child %d err %d\n", child, errno);
 
+	/* Put the tapdisk in a cgroup slice (best-effort) */
+	if (!slice) {
+#ifdef TAP_CTL_USE_DEFAULT_CGROUP_SLICE
+		/* No option specified; move it to the default slice */
+		tap_ctl_move_to_cgroup(id, "/");
```

Comment:
The comment here applies to the other invocation of this funciton as well.

Here we should check the return value and communicate how do we handle this situation.
So far, the invoked function has notified the error and passed the ball to the caller.
Here, according to your implementation, you decided to keep going without cgroups.
This decision needs to be communicated to the user.

In alternative, we can decide to abort.
This second scenario could be useful, for example, in case with a more recent kernel the new cgroups interface
will be used (I am not sure the old one will be kept in place).
In that case, if we silently continue we will spot the problem only when performance will degrade or worse.